torch_xla/__init__.py

```diff
@@ -243,6 +243,10 @@ def _init_xla_lazy_backend():
   plugins.use_dynamic_plugins()
   plugins.register_installed_plugins()
 
+if os.getenv('XLA_USE_EAGER_DEBUG_MODE', '0') == '1':
```

Comment:
Is this just for backward compatibility, since we already have the api now?

Reply:
yea, it is mostly for backward compatibility. I think I can also tweak it and use it in test to test both eager and non-eager.